jwql/utils/instrument_properties.py

```diff
@@ -234,3 +235,49 @@ def calc_frame_time(instrument, aperture, xdim, ydim, amps, sample_time=1.e-5):
             fullpad = 0
 
     return ((1.0 * xdim / amps + colpad) * (ydim + rowpad) + fullpad) * sample_time
+
+
+def get_obstime(filename):
+    """Extract the observation date and time from a fits file
+
+    Parameters
+    ----------
+    filename : str
+        Name of fits file
+
+    Returns
+    -------
+    obs_time : datetime.datetime
+        Observation date and time
+    """
+    with fits.open(filename) as h:
```

Comment:
@bhilbert4 I don't have much of an opinion on this one way or the other, but I wonder if it would be better to use `jwst.datamodels` here to extract this information?  Especially if it is faster than FITS IO.  What do you think?

Reply:
In general `jwst.datamodels` are much slower than `fits.io` because the datamodels do a bunch of metadata validation when the file is opened.